src/model/table.py

```diff
@@ -20,8 +40,49 @@ def set(self):
     def delete(self):
         pass
 
-    def get(self):
-        pass
+    def get(self, query):
```

Comment:
What do u mean by best_search ??!!!

Reply:
As I explained before.
- get most efficient index
- Get primary keys by this index if not null 
- Get objects by primary keys if not null otherwise, get all objects
- Filter those objects by query input